tests/test.py

```diff
@@ -439,11 +439,11 @@ def _test_release(self, package_name, data, dependency, main_repo=True):
         if main_repo:
             if dependency:
                 condition = (
-                    'base' in data
-                    and ('tags' in data or 'branch' in data)
-                    or ('sha256' in data
-                        and ('url' not in data
-                             or data['url'].startswith('http://')))
+                    'base' in data and
+                    ('tags' in data or 'branch' in data) or
+                    ('sha256' in data and
+                        ('url' not in data or
+                            data['url'].startswith('http://')))
```

Comment:
Using line breaks before binary operators is preferred. Please make new changes with the suggested style and don't change existing code.